tests/core/instruction/gates/test_gate.py

```diff
@@ -71,6 +71,7 @@ def test_power(gate: Gate, pow: float, result_matrix: npt.NDArray[np.complex64])
     assert np.allclose(gate.power(pow).to_matrix(), result_matrix)
 
 
+@pytest.mark.skip
```

Comment:
why skip this test?

Reply:
Because the feature of tensor product between gates is not fully implemented, so the test is ready but will not succeed, so we skip it

Reply:
I implemented the tensor product i think, so we can run this test now